types/office-js/index.d.ts

```diff
@@ -19071,6 +19071,164 @@ declare namespace Excel {
     function run<T>(context: OfficeExtension.ClientRequestContext, batch: (context: Excel.RequestContext) => Promise<T>): Promise<T>;
     function postprocessBindingDescriptor(response: any): any;
     function getDataCommonPostprocess(response: any, callArgs: any): any;
+    /**
+     * Contains information about a linked workbook.
+                If a workbook has links pointing to data in another workbook, the second workbook is linked to the first workbook.
+                In this scenario, the second workbook is called the "linked workbook".
+     *
+     * @remarks
+     * [Api set: ExcelApiOnline 1.1]
```

Comment:
Since the requirement sets are now generated in remarks for Excel (and soon the other application-specific APIs), we should look into making that consistent in Outlook and Common.

Reply:
Yeah, I was going to ask about this consistency issue too - maybe bring up in our team meeting